src/applications/appeals/995/tests/content/evidenceVaRecords.unit.spec.js

```diff
@@ -0,0 +1,13 @@
+import { expect } from 'chai';
+
+import { content } from '../../content/evidenceVaRecords';
+
+describe('title', () => {
+  it('should show title with add or edit and spelled out index', () => {
+    const { title } = content;
+    expect(title('add')).to.contain('Add the first VA facility');
+    expect(title('add', 3)).to.contain('Add the third VA facility');
+    expect(title('edit', 0)).to.contain('Edit the first VA facility');
+    expect(title('edit', 6)).to.contain('Edit the sixth VA facility');
```

Comment:
Still scratching my head how the first VA facility is at index 0 but sixth VA facility is at index 6.

Reply:
See previous comment about using a one-based index for screen reader content.